assets/_sass/custom-viz/boeing/_base.scss

```diff
@@ -82,10 +88,11 @@ $size_boeing-container-max-width: 700px;
 
     .part {
       margin: 0.1rem;
-      background-color: $yellow;
+      border-bottom: 1px solid $yellow;
 
       &:hover {
-        background-color: $dark-yellow;
+        color: $dark-gray;
+        background-color: $yellow;
         transition: all 0.2s ease-in-out;
```

Comment:
Remember that transitions need to be applied to the base class, not the `:hover` state.